src/TransitionSystems/quotient.rs

```diff
@@ -173,7 +173,7 @@ impl TransitionSystem for Quotient {
             return transitions;
         } else if location.is_universal() {
             // Rule 9
-            let transition = Transition::new(location, self.dim);
+            let transition = Transition::new(TransitionID::Simple(format!("U|Universal:{}", location.id)), location, self.dim);
```

Comment:
Again, is it intentional that this is different than in compiled components .rs?

Reply:
Will be removed after debug session